tests/testthat/test-mc.R

```diff
@@ -72,6 +72,9 @@ test_that("strata", {
 
 
 test_that("bad args", {
+  expect_snapshot(error = TRUE, {
+    mc_cv(mtcars, prop = 1)
+  })
```

Comment:
```suggestion
  expect_snapshot(error = TRUE, {
    mc_cv(mtcars, prop = -1)
  })
  expect_snapshot(error = TRUE, {
    mc_cv(mtcars, prop = 1)
  })
```

...as we haven't checked the "greater than 0" message.